src/Types/Results.ts

```diff
@@ -36,6 +36,7 @@ export type Program = {
   already_has: boolean;
   new: boolean;
   navigators: ProgramNavigator[];
+  documents: Translation[];
```

Comment:
Thanks for the change. I know where to find documents now. Just one question, I guess I have to wait for the backend change get merged first then I can see this new field, right? Thanks.

Reply:
Yes, that change will be merged this Friday.